internal/fileops/file_ops_test.go

```diff
@@ -380,31 +369,25 @@ func TestSyncDir(t *testing.T) {
 }
 
 func prepareTestDir(t *testing.T) string {
-	tempDir, err := ioutil.TempDir(os.TempDir(), "UnitTest-fileops")
-	require.NoError(t, err)
-
-	removeAndFailNow := func() {
-		os.RemoveAll(tempDir)
-		t.FailNow()
-	}
+	tempDir := t.TempDir()
 
 	if err := os.Mkdir(path.Join(tempDir, "dir"), 0755); err != nil {
-		removeAndFailNow()
+		t.FailNow()
 	}
 	if err := os.Mkdir(path.Join(tempDir, "dir", "a"), 0755); err != nil {
-		removeAndFailNow()
+		t.FailNow()
 	}
 	if err := os.Mkdir(path.Join(tempDir, "dir", "b"), 0755); err != nil {
-		removeAndFailNow()
+		t.FailNow()
 	}
 	if err := os.Mkdir(path.Join(tempDir, "dir", "c"), 0755); err != nil {
-		removeAndFailNow()
+		t.FailNow()
 	}
 	if err := os.Mkdir(path.Join(tempDir, "dir", "d"), 0755); err != nil {
-		removeAndFailNow()
+		t.FailNow()
```

Comment:
Can we use here `require.NoError(t, ...)` instead of `if err := ...; err != nil {t.FailNow()}`?

Reply:
Sure :ok_hand: